docs/FAQs.md

```diff
@@ -47,3 +47,91 @@ To enable `forceAjcCompile` you need to use following `aspectj-maven-plugin` con
 </configuration>
 ```
 
+## How can I use Powertools for AWS Lambda (Java) with the AWS CRT HTTP Client?
+
+Powertools uses the `url-connection-client` as the default http client. The `url-connection-client` is a lightweight http client, which keeps the impact on Lambda cold starts to a minimum. 
+With the [announcement](https://aws.amazon.com/blogs/developer/announcing-availability-of-the-aws-crt-http-client-in-the-aws-sdk-for-java-2-x/) of the `aws-crt-client` a new http client has been released, which offers faster SDK startup time and smaller memory footprint. 
+
+Unfortunately, replacing the `url-connection-client` dependency with the `aws-crt-client` will not immediately improve the lambda cold start performance and memory footprint, 
+as the default version of the dependency contains low level libraries for all target runtimes (linux, macos, windows, etc).  
+
+Using the `aws-crt-client` in your project requires the exclusion of the `url-connection-client` transitive dependency from the powertools dependency. 
+
+```xml 
+<dependency>
+    <groupId>software.amazon.lambda</groupId>
+    <artifactId>powertools-parameters</artifactId>
+    <version>1.18.0</version>
+    <exclusions>
+        <exclusion>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>url-connection-client</artifactId>
+        </exclusion>
+    </exclusions>
+</dependency>
+```
+Next to adding the `aws-crt-client` and excluding the `aws-crt` dependency (contain all runtime libraries) it's required to set a specific runtime version of the `aws-crt` dependency by specifying the classifier for your specific target runtime.
+Specifying the specific target runtime makes sure all other target runtimes are excluded from the jar file, which will result in the benefit of improved cold start times.   
```

Comment:
Understood, then maybe rephrase:
```suggestion
By specifying the specific target runtime, it will avoid other target runtimes to be included in the lambda jar file, which will result in a smaller package and the benefit of improved cold start times.   
```